code/script.js

```diff
@@ -1,19 +1,133 @@
-// Start here
+//Slows down the first alert to make everything on the site have time to load before the alert appears
+window.onload = function () { 
+  setTimeout(function() {
 
 // Step 1 - Welcome and introduction
-// Your code goes here
+//An alert with a welcoming message
 alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+  `Welcome to our Pizzeria The Cheesy Slice! Ready to Start? - Click 'OK' to begin.`
+)
+
+//Using the prompt method to ask for the name of the customer and an alert uses that variable to greet the customer
+const guestName = prompt(`Dear guest, what is your name?`)
+console.log(guestName)
```

Comment:
I left them since it was ok up until our last project to leave them in the code according to our code coach. However, I agree that it's looks cleaner without them so I have removed them now!